project.yml

```diff
@@ -0,0 +1,28 @@
+name: SnapshotTesting
```

Comment:
Here's the main project.yml file. Wasn't so bad once I saw the `$platform` trick!